pagefaultgames/pokerogue
# [Bug] Fix crash when using enemy move override

## What are the changes the user will see?
Potentially some missing animation issues will no longer cause a crash... or it's possible the issue is only caused specifically by the override.

## Why am I making these changes?
It's really annoying when the game crashes while testing. ![](https://cdn.discordapp.com/emojis/1257794821175640196.webp?quality=lossless)

## What are the changes from a developer perspective?
The game will crash less when using the enemy moveset override.

### Screenshots/Videos
Before fix:
![image](https://github.com/user-attachments/assets/bd5402ec-d5dc-4986-9eb8-4ff6c965b6da)

## How to test the changes?
```ts
const overrides = {
  OPP_MOVESET_OVERRIDE: Moves.SPLASH
} satisfies Partial<InstanceType<typeof DefaultOverrides>>;
```
Start a new Classic run with any random Pokémon, make sure not to faint the enemy on the first turn.

## Checklist
